client/less/main.less

```diff
@@ -271,10 +271,11 @@ h1, h2, h3, h4, h5, h6, p, li {
 
 a {
   font-weight: bold;
+  font-size : 15px;
 }
 
 p {
-  font-size: 16px;
+  font-size: 15px;
 }
 
```

Comment:
@AbdelghaniTassi Great job! I noticed that the tests are still 14px:

![regular_expressions__match_literal_strings___freecodecamp](https://user-images.githubusercontent.com/985197/29598155-b4a271c6-878c-11e7-8b85-ab797c4b75d9.png)

Can you also make these 15px for consistency?